operator/internal/manifests/openshift/rbac.go

```diff
@@ -100,6 +100,14 @@ func BuildRulerClusterRole(opts Options) *rbacv1.ClusterRole {
 					"patch",
 				},
 			},
+			{
+				NonResourceURLs: []string{
+					"/api/v2/alerts",
+				},
+				Verbs: []string{
+					"create",
+				},
+			},
```

Comment:
Can you explain a bit why this is needed for the user-workload-monitoring Alertmanager? What is the above patch rbac not enough as in the cluster-monitoring case?

Reply:
I don't know how to explain it other than: If you don't have this permission, the user worload SVC throws a 403 in your face :smiling_face_with_tear:

Reply:
I guess it would good to ask the monitoring team on why the apply two different approaches. Maybe the will converge to the new one, who knows. Anyway out of curiosity it might good to know.